example/TabNavigator.tsx

```diff
@@ -143,7 +144,7 @@ const TabRouter: Router<Action | CommonAction> = {
   },
 };
 
-export default function TabNavigator(props: Props) {
+export default createNavigator((props: Props) => {
```

Comment:
I think better to do `export default createNavigator(TabNavigator)` below so this component has a display name. Otherwise it'll show up as `Unnamed` in dev tools and `anonymous` in stack traces.